### PR TITLE
indic tts without pretrained model

### PR DESCRIPTION
**Pull request by peter-yh-wu**

### CODE REVIEW

File: egs2/cmu_indic/tts1/sid [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../tools/kaldi/egs/sre08/v1/sid
```
Comment (ftshijt) [inline egs2/cmu_indic/tts1/sid:1]: Is this part used somewhere?

File: egs2/cmu_indic/tts1/local/data_prep.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+paste -d " " <(echo "${ids}") <(echo "${sentences}") > ${text}.tmp
+local/clean_text.py ${text}.tmp > ${text}
+rm ${text}.tmp
+echo "Successfully finished making text."
+
+utils/fix_data_dir.sh data/${spk}
+utils/validate_data_dir.sh --no-feats data/${spk}
+
+# split
+utils/subset_data_dir.sh --last data/${spk} 200 data/${spk}_tmp
```
Comment (ftshijt) [inline egs2/cmu_indic/tts1/local/data_prep.sh:81]: Just to make sure: is the split offer officially somewhere?

**Review by ftshijt — COMMENTED** [review]
Please also add the corpus entry at `egs2/README.md`

File: egs2/cmu_indic/tts1/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# CMU INDIC RECIPE
+
+This is the recipe of the TTS model with the [CMU INDIC](http://festvox.org/cmu_indic/) databases.
```
Comment (kan-bayashi) [inline egs2/cmu_indic/tts1/README.md:3]:
```suggestion
This is the recipe of the TTS model with the [CMU INDIC](http://festvox.org/cmu_indic/) databases.

See the following pages for the usage:
- [How to run the recipe](../../TEMPLATE/tts1/README.md#how-to-run)
- [How to train FastSpeech](../../TEMPLATE/tts1/README.md#fastspeech-training)
- [How to train FastSpeech2](../../TEMPLATE/tts1/README.md#fastspeech2-training)

See the following pages before asking the question:
- [ESPnet2 Tutorial](https://espnet.github.io/espnet/espnet2_tutorial.html)
- [ESPnet2 TTS FAQ](../../TEMPLATE/tts1/README.md#faq)
```

File: egs2/cmu_indic/tts1/local/clean_text.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
```
Comment (kan-bayashi) [inline egs2/cmu_indic/tts1/local/clean_text.py:1]: For espnet2, text cleaning will be performed on the fly.
So you do not need it.

File: egs2/cmu_indic/tts1/local/data_prep.sh [hunk 5]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Tomoki Hayashi
```
Comment (kan-bayashi) [inline egs2/cmu_indic/tts1/local/data_prep.sh:3]:
```suggestion
# Copyright 2021 Peter Wu
```

File: egs2/cmu_indic/tts1/local/data_prep.sh [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+utils/utt2spk_to_spk2utt.pl ${utt2spk} > ${spk2utt}
+echo "Successfully finished making spk2utt."
+
+# make text
+raw_text=${db}/etc/txt.done.data
+ids=$(sed < ${raw_text} -e "s/^( /${spk}_/g" -e "s/ )$//g" | cut -d " " -f 1)
+sentences=$(sed < ${raw_text} -e "s/^( //g" -e "s/ )$//g" -e "s/\"//g" | tr '[:lower:]' '[:upper:]' | cut -d " " -f 2-)
+paste -d " " <(echo "${ids}") <(echo "${sentences}") > ${text}.tmp
+local/clean_text.py ${text}.tmp > ${text}
+rm ${text}.tmp
```
Comment (kan-bayashi) [inline egs2/cmu_indic/tts1/local/data_prep.sh:74]: For espnet2, text cleaning will be performed on the fly.
So you do not need to perform cleaning here. Just make `text` with raw transcription.

File: egs2/cmu_indic/tts1/conf/train.yaml [hunk 7]
```diff
@@ -0,0 +1 @@
+tuning/train_gst_tacotron2.yaml
```
Comment (kan-bayashi) [inline egs2/cmu_indic/tts1/conf/train.yaml:1]: Is this recipe single speaker or multi speaker?
If single speaker, we do not need to use xvector, please remove x-vector-based configs.

FYI: Placing minimum configurations is fine.
https://github.com/espnet/espnet/tree/master/egs2/ruslan/tts1/conf/tuning

File: egs2/cmu_indic/tts1/run.sh [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+./tts.sh \
+    --lang en \
+    --feats_type raw \
+    --fs "${fs}" \
+    --n_fft "${n_fft}" \
+    --n_shift "${n_shift}" \
+    --win_length "${win_length}" \
+    --token_type phn \
+    --cleaner tacotron \
```
Comment (kan-bayashi) [inline egs2/cmu_indic/tts1/run.sh:35]: FYI: This perform the same thing of `local/clean_text.py`.

File: egs2/cmu_indic/tts1/sid (hunk 1 above)
Comment (kan-bayashi) [reply egs2/cmu_indic/tts1/sid:1]: This is used to extract x-vector. Only used for multi-speaker case.
(This symlink is included in TEMPLATE.)

**Review by kan-bayashi — COMMENTED** [review]
Thanks for your contributions, Peter.
Please check my comments.

**Review by kan-bayashi — APPROVED** [review]
LGTM

**Pull request merged**